packages/logger/package.json

```diff
@@ -47,6 +47,10 @@
     "./types": {
       "import": "./lib/esm/types/index.js",
       "require": "./lib/cjs/types/index.js"
+    },
+    "./correlationId": {
+      "import": "./lib/esm/correlationId.js",
+      "require": "./lib/cjs//correlationId.js"
```

Comment:
Double `//` int the path, plus you're missing the types in the `typesVersions` object below at L56 - if you build the pkg and import it, it'll not find this sub-path export.